lib/rspec/rails/view_rendering.rb

```diff
@@ -44,6 +44,13 @@ def render_views?
       #
       # @private
       class EmptyTemplateResolver < ::ActionView::FileSystemResolver
+
+        def initialize(path, pattern=nil)
+          unless path.is_a?(::ActionView::Resolver)
+            super
+          end
+        end
+
```

Comment:
Yes, but it'd generate template missing errors because `find_templates` would be called on the `EmptyTemplateResolver` instance and not the custom resolver instance.